lib/ecto/adapters/postgres/connection.ex

```diff
@@ -187,6 +187,7 @@ if Code.ensure_loaded?(Postgrex) do
 
     defp conflict_target({:constraint, constraint}),
       do: ["ON CONSTRAINT ", quote_name(constraint), ?\s]
+    defp conflict_target({:fragment, fragment}), do: [fragment, ?\s]
```

Comment:
(Minor nitpick) This is a personal style preference: I would align the `do:` of this expression with the `do:` of the other expressions:

```elixir
    defp conflict_target({:constraint, constraint}),
      do: ["ON CONSTRAINT ", quote_name(constraint), ?\s]
    defp conflict_target({:fragment, fragment}), 
      do: [fragment, ?\s]
    defp conflict_target([]),
      do: []
    defp conflict_target(targets),
      do: [?(, intersperse_map(targets, ?,, &quote_name/1), ?), ?\s]
```